### PR TITLE
Refactor Payment System.

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: golem/transactions/paymentskeeper.py [hunk 1]
```diff
@@ -70,117 +62,33 @@ def get_newest_payment(num=30):
...
         self.db.add_payment(payment_info)
-        task.subtasks[payment_info.subtask_id] = SubtaskPaymentInfo(payment_info.value, payment_info.computer)
-        task.value += payment_info.value
-        if payment_info.task_id in self.finished_tasks:
-            self.finished_tasks.remove(payment_info.task_id)
-            self.__put_task_in_tasks_to_pay(payment_info.task_id)
 
     def load_from_database(self):
-        return [{"task": payment.task, "node": payment.to_node_id, "value": payment.val, "state": payment.state} for
+        return [{"subtask": payment.subtask,
```
Comment (badb) [inline golem/transactions/paymentskeeper.py]: It brokes the GUI, which is expecting "task".

File: golem/model.py [hunk 2]
```diff
@@ -1,116 +1,127 @@
...
+    payee = CharField()
+    value = IntegerField()
+    details = CharField(null=True)
+
+
+class ReceivedPayment(BaseModel):
+    """ Represent payments that nodes on this machine receive from other nodes
+    """
+    from_node_id = CharField()
+    task = CharField()
```
Comment (badb) [inline golem/model.py:122]: Shouldn't we change this in "subtask" too?

File: golem/transactions/paymentskeeper.py (hunk 1 above)
Comment (chfast) [inline golem/transactions/paymentskeeper.py]: Ok, I will revert that. UI changes are planned after the Payment System is working.

File: golem/model.py (hunk 2 above)
Comment (chfast) [inline golem/model.py:122]: I will take care of incoming payments later on. This change is all about outgoing payments.

**Pull request merged**